src/Compilers/Core/Portable/CommandLine/CommonCompiler.cs

```diff
@@ -130,10 +140,10 @@ internal virtual MetadataReferenceResolver GetCommandLineMetadataReferenceResolv
         /// <param name="file">Source file information.</param>
         /// <param name="diagnostics">Storage for diagnostics.</param>
         /// <returns>File content or null on failure.</returns>
-        internal SourceText ReadFileContent(CommandLineSourceFile file, IList<DiagnosticInfo> diagnostics)
+        internal SourceText TryReadFileContent(CommandLineSourceFile file, IList<DiagnosticInfo> diagnostics)
```

Comment:
Since there precedence for this pattern and it was @tmat's suggestion from his earlier review, I'm going to stick with `Try`